tests/testthat/_snaps/shuffle_ard.md

```diff
@@ -0,0 +1,58 @@
+# shuffle/trim works
```

Comment:
For all the snapshot tests, can we add a `as.data.frame()`? The data frame print method shows all rows and columns, but the tibble truncates. Also, I've seen tibble make small updates to their print method over the years, which will break the snapshots, while data frame's print method is super stable.

Reply:
yes that's super smart!